src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -164,6 +168,10 @@
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
+
+  &.banned {
+    opacity: 0.2;
+  }
 }
```

Comment:
I think there should be a indicator that a participant is banned instead of lowering the opacity. Setting the opacity to 20% for banned participants could be problematic if you are thinking of people with visual impairments, for example.

Reply:
While I agree, I think @SelinaBuff should decide it.
However, I added a title on the ban/unban button which references the user name, so that it should be possible for screen readers to tell that a specific user is banned and can be unbanned.